_posts/2020-08-01-SORSE-Call-For-Contributions.markdown

```diff
@@ -0,0 +1,21 @@
+---
+layout: post
+title:  "SORSE Call for Contributions"
+date:   2020-08-01 12:00:00 +1000
+categories: online events 2020
+---
+
+A global collaboration has established the [*International Series of Online Research Software Events*](https://sorse.github.io/), aka SORSE (pronounced 'sauce'). The goal is to provide an opportunity for RSEs to develop and grow their skills, build new collaborations and engage with RSEs worldwide. 
+
+<a class="rse rse-join" target="_blank" href="https://sorse.github.io/programme/call-for-contributions/">Call for Contributions</a>
+
+You can make contributions, about technical or soft skills, in any of the following formats:
+
+- [Panels and Discussions](https://sorse.github.io/programme/panels/)
+- [Posters and Lightning talks](https://sorse.github.io/programme/posters/)
+- [Software demos](https://sorse.github.io/programme/software-demos/)
+- [Talks](https://sorse.github.io/programme/talks/)
+- [Workshops](https://sorse.github.io/programme/workshops/)
+- [Blog Posts](https://sorse.github.io/faq/howto/contribute-blog-posts)
+
+This is an open continuous call with rolling deadlines at the end of the last day (UTC) of each month, and the next deadline is set for 1st September.
```

Comment:
Probably should be 1st September

Reply:
The deadline date keeps becoming stale. Perhaps, we should just take the actual date out, and say 1st of the month

Reply:
Removed specific deadline.